demos/tabs.html

```diff
@@ -587,8 +641,12 @@ <h1 class="mdc-toolbar__title">Title</h1>
     <script>
         (function() {
           document.getElementById('toggle-rtl').addEventListener('click', function(evt) {
-            evt.target.parentElement.hasAttribute('dir') ? evt.target.parentElement.removeAttribute('dir') : evt.target.parentElement.setAttribute('dir', 'rtl');
-            window.basicTabBar.layout()
+            var mainElement = document.querySelector('#tabs-demo-main-section');
+            mainElement.hasAttribute('dir') ? mainElement.removeAttribute('dir') : mainElement.setAttribute('dir', 'rtl');
+
+            window.basicTabBar.layout();
+            window.tabBarScroller.layout();
+            tabBarScroller.tabBar.layout();
```

Comment:
nit: why not reference tabBarScroller using `window` here since it's referenced using `window` everywhere else?